cmd/buildctl/diskusage.go

```diff
@@ -25,7 +27,33 @@ func diskUsage(clicontext *cli.Context) error {
 		return err
 	}
 
-	log.Printf(spew.Sdump(du))
+	tw := tabwriter.NewWriter(os.Stdout, 1, 8, 1, '\t', 0)
+
+	fmt.Fprintln(tw, "ID\tRECLAIMABLE\tSIZE\tLAST ACCESSED")
```

Comment:
Yes, not yet. The backend doesn't support it yet but added as a hint of what fields should be here.